components/sysutils/stunnel/Makefile

```diff
@@ -21,31 +21,32 @@
 
 #
 # Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
+# copyright (c) 2021, Tim Mooney.  All rights reserved.
 #
 
+BUILD_STYLE=            configure
+BUILD_BITS=             64
```

Comment:
BUILD_STYLE= configure is the default so you don't need to set it here.

Reply:
Dropped `BUILD_STYLE`.   It may have been inserted automatically by `gmake update`